dozer/cogs/info.py

```diff
@@ -10,48 +12,85 @@
-            e.add_field(name='Bot Created' if member.bot else 'User Joined Discord',
-                        value=member.created_at.strftime(datetime_format))
-            e.add_field(name='Joined Guild', value=member.joined_at.strftime(datetime_format))
-            e.add_field(name='Color', value=str(member.color).upper())
-
-            e.add_field(name='Status and Game', value=f'{member.status}, '.title() + (
-                f'{activity_name}' + f' {member.activity.name}' if member.activity else 'no game playing'), inline=False)
-            roles = reversed(member.roles[1:])  # Remove @everyone
-            e.add_field(name='Roles', value=', '.join(role.name for role in roles) or "No roles", inline=False)
-            e.add_field(name='Icon URL', value=icon_url, inline=False)
-        await ctx.send(embed=e)
+    @bot_has_permissions(embed_links=True)
+    async def member(self, ctx, *, member: discord.Member = None):
+        """ Show information on a member of this guild.
+            If no member is specified, your information will be shown."""
+        if member is None:
+            member = ctx.author
+
+        icon_url = member.avatar_url_as(static_format='png')
+
+        embed = discord.Embed(title=member.display_name, description=f'{member!s} ({member.id})', color=member.color)
+        embed.add_field(name='Bot Created' if member.bot else 'Account Created',
+                        value=member.created_at.strftime(self.datetime_format), inline=True)
+        embed.add_field(name='Member Joined', value=member.joined_at.strftime(self.datetime_format), inline=True)
+        if member.premium_since is not None:
+            embed.add_field(name='Member Boosted', value=member.premium_since.strftime(self.datetime_format), inline=True)
+        embed.add_field(name='Color', value=str(member.color).upper(), inline=True)
+
+        status = 'DND' if member.status is discord.Status.dnd else member.status.name.title()
+        if member.status is not discord.Status.offline:
+            platforms = self.pluralize([platform for platform in ('web', 'desktop', 'mobile') if
+                                        getattr(member, f'{platform}_status') is not discord.Status.offline])
+            status = f'{status} on {platforms}'
+        activities = '\n'.join(self._format_activities(member.activities))
+        embed.add_field(name='Status and Activity', value=f'{status}\n{activities}', inline=True)
+
+        embed.add_field(name='Roles', value=', '.join(role.name for role in member.roles[:0:-1]) or 'None')
+        embed.add_field(name='Icon URL', value=icon_url)
+        embed.set_thumbnail(url=icon_url)
+        await ctx.send(embed=embed)
 
     member.example_usage = """
-    `{prefix}member` - get information about yourself
-    `{prefix}member cooldude#1234` - get information about cooldude
-    """
+        `{prefix}member`: show your member info
+        `{prefix}member {ctx.me}`: show my member info
+        """
```

Comment:
This indentation is inconsistent with other example usages.

Reply:
Oops